tutorials/getting-started/layout-widgets.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Layout (nested) widgets"
+layout: tutorial
+---
+
+## Layout (nested) widgets
+
+You’ve built a couple of custom widgets, you’ve got a powerful `area` running down the main column of your template, awesome. Now you want start pairing widgets up, creating complex layouts of widgets. `singletons` let you enforce particular widgets in particular pieces of layout, but you want the editor to decide where it's appropriate to create these complex arrangements and be able to mix them with simpler widgets.
+
+You want a widget that creates a layout of inner widgets.
+
+### Create a layout widget
+
+Generally, you need a dedicated widget to create layout. The Apostrophe team sometimes refers to these as ‘layout widgets’ that give form and structure to ‘content widgets’ (widgets whose sole job is to display content). **In this example we'll make a simple two column layout.**
+
+With the `apostrophe-cli` installed
+
+```bash
+apostrophe create-widget two-column
+```
+
+We'll include the layout widget in our `app.js` by adding the following to the `modules` object:
+
+```javascript
+  modules: {
+    // ...,
+    'two-column-widgets': {}
+  }
+```
+
+Now we'll define what can be in a `two-column` widget by defining it in `lib/modules/two-column-widgets/index.js`:
+
+```javascript
+module.exports = {
+  extend: 'apostrophe-widgets',
+  label: 'Two Column Layout',
+  // contextualOnly: true,
+  // skipInitialModal: true
```

Comment:
This second one is redundant if we have contextualOnly.

Reply:
This is what I was referencing above .. How do I point out where these properties go without duplicating (and possibly losing someone who copy-pastes the first code block) code? Should I just leave them out of the initial example and then provide truncated examples when each prop is explained?

Reply:
I would include the one you almost certainly want, fully activated, in the initial example (contextualOnly), and then give truncated examples or just say "similar to the way contextualOnly is used above, you can also use..."

Reply:
yeah i did something like this in the edit https://github.com/punkave/apostrophe-documentation/commit/366f934974f3d007c598316ee06d0fe78430240e#diff-cf5f835f95f84c963b57555d91853bfdR37